nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/PropertyCommentManager.java

```diff
@@ -131,8 +148,9 @@ public DocumentModel createComment(DocumentModel docModel, DocumentModel comment
                     (Serializable) computeAncestorIds(session, docModel.getId()));
             DocumentModel comment = session.createDocument(commentModelToCreate);
             comment.detach(true);
-            notifyEvent(session, CommentEvents.COMMENT_ADDED, docModel, comment);
 
+            handleNotificationAutoSubscriptions(session, commentModel, session.getDocument(docRef));
+            notifyEvent(session, CommentEvents.COMMENT_ADDED, comment);
```

Comment:
why we `handleNotificationAutoSubscriptions` and `notifyEvent` on the `Get` we just retrieve why we need to notify an event

Reply:
The notification was already there but with the 4 args signature.
This one is the 3 args version. We didn't use it the Tree but I don't remember it was rolledback on Property as well ?

Reply:
If it was here before i think we can compare to `TreeCommentManager`, we should investigate on it to see why we did it in the past ... just to understand the things better. For the `3` vs `4` args as i explained earlier when we have the `commentedDocumentModel` we made the choice (with Kevin) to keep calling the `4` args one just to avoid reloading things that we already have

Reply:
my bad we told about the `CreateComment` not the `Get`, anyway we should use the version with `4` parameters as it is on master

Reply:
I put back the 4 args version 👍

Reply:
thx, when said `Also did in createLocatedComment l:187` -> you mean as it is on master ?

Reply:
Yes reverted it